src/components/Dialog/Dialog.css

```diff
@@ -51,12 +51,12 @@
   width: 432px;
   height: 574px;
   top: 2.5%;
-  left: 25%;
-  right: 25%;
+  left: 0;
+  right: 0;
   bottom: 0%;
   background-color: #fff;
   z-index: 3;
-  margin: 25px;
+  margin: auto;
```

Comment:
Just checking: what is the purpose of setting this `auto` here?

Reply:
`margin: auto` is what is centering this content. Despite having an `absolute` position and all the left, top, right, and bottom set to `0`, the content will not be centered unless the margin is set to `auto`. This horizontally and vertically centers the content.